src/suparnaNJune20/CodingExam/SecondMaxNo.java

```diff
@@ -0,0 +1,21 @@
+package suparnaNJune20.CodingExam;
+public class SecondMaxNo {
+
+	void fndSecondMaxNo(int number[]) {
+		int max1No = number[0], max2No = number[0];
+		for (int outerIndex = 0; outerIndex < number.length; outerIndex++) {
+			if (number[outerIndex] > max1No && number[outerIndex] > max2No) {
```

Comment:
here don;t need to add condition for max2No, because anyways if current number is greater than max number then it is always greater than secondmax number